packages/core/ios/RNEmbrace/EmbraceManager.swift

```diff
@@ -329,6 +333,49 @@ class EmbraceManager: NSObject {
         
     }
     
+    @objc(startView:resolver:rejecter:)
+    func startView(_ viewName: String, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
+        let startTimeDate = Date()
```

Comment:
I think this would be cleaner to call the `Embrace.client` methods for creating a span directly rather than the `startSpan` bridge methods which do some extra work, something like:
```
    let span = Embrace.client?.buildSpan(name: name).startSpan()
    // reject if span is nil

    span?.setAttribute(key: key, value: value)     // set ux.view, etc.
    Embrace.client?.flush(span!)

    let id = spanRepository.spanStarted(span: span!)
    // reject if id.isEmpty
    resolve(id)
```

Reply:
actually the builder supports setting attributes so could save the flush call with something like:
```
let span = Embrace.client?.buildSpan(name: name)
                 .setAttribute(key:value:)
                 .setAttribute(key:value:)
                 .startSpan()
```